app/src/main/java/com/example/moviews/data/MovieDataSource.kt

```diff
@@ -9,11 +9,13 @@ interface MovieDataSource {
         fun deleteMovie(movieId: String, callback: OnLoadDataCallback<Boolean>)
         fun getAllMovies(callback: OnLoadDataCallback<MutableList<Movie>>)
     }
-    interface Remote{
+
+    interface Remote {
         fun getTrendingMovie(callback: OnLoadDataCallback<MutableList<Movie>>)
         fun getUpcomingMovie(callback: OnLoadDataCallback<MutableList<Movie>>)
         fun getNowPlayingMovie(callback: OnLoadDataCallback<MutableList<Movie>>)
         fun getPopularMovie(callback: OnLoadDataCallback<MutableList<Movie>>)
         fun getTopSearchMovie(callback: OnLoadDataCallback<MutableList<Movie>>)
+        fun getMovieByGenre(idGenre: Int, callback: OnLoadDataCallback<MutableList<Movie>>)
```

Comment:
```suggestion
        fun getMovieByGenres(idGenre: Int, callback: OnLoadDataCallback<MutableList<Movie>>)
```
hình như mấy chỗ trên nữa